examples/react/ui/src/ContentPage.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import { SimpleUploader, W3Upload } from '@w3ui/react-ui'
+import { Uploader, UploaderProvider, useUploader } from '@w3ui/react-uploader'
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
+import { a11yDark } from 'react-syntax-highlighter/dist/esm/styles/prism'
+
+function NoUIUploadComponent () {
+  const [{ storedDAGShards }, uploader] = useUploader()
+
+  return (
+    <div>
+      <input type='file' onChange={e => uploader.uploadFile(e.target.files[0])} />
+      {storedDAGShards?.map(({ cid, size }) => (
+        <p key={cid.toString()}>
+          {cid.toString()} ({size} bytes)
+        </p>
+      ))}
+    </div>
+  )
+}
+
+function NoUIComponent () {
+  return (
+    <UploaderProvider>
+      <NoUIUploadComponent />
+    </UploaderProvider>
+  )
+}
+
+function HeadlessUIComponent () {
+  return (
+    <UploaderProvider>
+      <Uploader>
+        <Uploader.Form>
+          <div>
+            <label htmlFor='file'>File:</label>
+            <Uploader.Input id='file' />
+          </div>
+          <button type='submit'>Upload</button>
+        </Uploader.Form>
+      </Uploader>
+    </UploaderProvider>
+  )
+}
+
+function CustomizableUIComponent () {
+  return (
+    <UploaderProvider>
+      <SimpleUploader />
+    </UploaderProvider>
+  )
+}
+
+function DropinUIComponent () {
+  return (
+    <W3Upload />
+  )
+}
+
+export function ContentPage () {
+  return (
+    <>
+      <h1 className='title'>w3ui UI Components</h1>
+      <p>
+        w3ui provides React components at four different levels of
+        abstraction, each of which builds upon the last. In order they are:
+      </p>
+      <h3>1. No UI</h3>
+      <p>
+        Maximum flexibility, React Contexts and hooks only:
+      </p>
+      <SyntaxHighlighter language='jsx' style={a11yDark}>
+        {`
+function NoUIUploadComponent() {
```

Comment:
Ooh! Could toString on the existing functions here to avoid having it twice!

```suggestion
${NoUIUploadComponent.toString()}
```